cypress/integration/plugins/observability-dashboards/3_trace_analytics_traces.spec.js

```diff
@@ -55,7 +55,7 @@ describe('Testing traces table', () => {
     cy.get('input[type="search"]').focus().type(`${TRACE_ID}{enter}`);
     cy.get('.euiButton__text').contains('Refresh').click();
     cy.contains(' (1)').should('exist');
-    cy.contains('03/25/2021 10:21:22').should('exist');
+    cy.contains('03/25/2021 17:21:22').should('exist');
```

Comment:
In which timezone the data is injected? 
How is it working with daylight savings in the current scenario?